rqt/setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup
+
+package_name = 'rqt'
+setup(
+    name=package_name,
+    version='0.5.0',
+    install_requires=['rqt_gui', 'rqt_gui_cpp', 'rqt_gui_py', 'rqt_py_common'],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
```

Comment:
is this actually needed? launch is a pure python package and it doesn't do this empty file thing

Reply:
You're right, this is no longer needed it seems.